autogpt_platform/backend/.env.default

```
@@ -90,6 +90,7 @@ MEDIA_GCS_BUCKET_NAME=
 OPENAI_API_KEY=
 ANTHROPIC_API_KEY=
 GROQ_API_KEY=
+GOOGLE_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GOOGLE_API_KEY key should go before the GROQ_API_KEY key

(UnorderedKey)